package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ship-components-utility",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
This should probably be 1.1.0 according to the [semver](http://semver.org/) spec. So, let's switch that.

> MAJOR version when you make incompatible API changes,
> MINOR version when you add functionality in a backwards-compatible manner, and
> PATCH version when you make backwards-compatible bug fixes.
> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.